internal/tools/seed.go

```diff
@@ -0,0 +1,42 @@
+package tools
+
+import (
+	"github.com/broadinstitute/sherlock/internal/db"
+	"github.com/broadinstitute/sherlock/internal/services"
+)
+
+// SeedServices is a test utility that will populate a database with a predetermined list of "services"
+// to be used for running integration tests against a real database
+func SeedServices(repository *db.Repository) ([]services.Service, error) {
```

Comment:
I see a couple places where the `Repository`/database client is passed in as a parameter where I'd either expect either (a) the function to use a `Repository` as a receiver or (b) the `Repository`/database to follow the singleton pattern, kept in global state* 🤔 This parameter passing seems a bit verbose coming from Ruby.

*I know this can be controversial haha. This blog post was pretty interesting: https://alexedwards.net/blog/organising-database-access

Reply:
And I know the "BUT THIS IS HOW RUBY DOES ITTTT" comments must be getting old 😁

Reply:
Thank you for calling me out on this. I was actually basing my database access patterns on section 3 from that very article. However I did forget to go back and turn the methods that accept because I got distracted by the gin/gorm stuff  so thank you for bringing attention to this. 

There is one go quirk that comes into play here. You can only define receiver methods in the same `package` where the receiver type is defined. So that would not be doable in the case of these seeding helper functions. 

I would like to keep some separation with these specific methods as they are only utilities for integration test setup and not part of core application logic. So I think this particular file should stay as it is. 

I will go back and update the other functions in the `db` package that use this same pattern to take a receiver instead though :).

Reply:
Also agree `repository` isn't the best name for the type. That was just a convention I saw online and was having trouble thinking of a good alternative..

Reply:
> You can only define receiver methods in the same package where the receiver type is defined. 
Ahhh... TIL! In retrospect this makes a lot of sense.

Thanks for the clarification! 👍